docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -0,0 +1,466 @@
+
+- Inviting a customer to come back to the site by pushing a message with recommendations, when they haven't returned to the site for a set time.
+
+- Reviving the customer's interest by pushing a message with products that are similar to the ones the customer has already seen.
+
+- Inducing a purchase by pushing a message when a price of the product from the customer's wishlist decreases.
+
+For more information, see [Email triggers](https://doc.ibexa.co/projects/userguide/en/4.5/personalization/triggers.md).
+
+#### Multiple attributes in recommendation computation
+
+With this feature, you get an option to combine several attribute types when computing recommendations.
+As a result, users can be presented with recommendations from an intersection of submodel results.
+
+For more information, see [Submodel parameters](recommendation_api.md#submodel-parameters) and [Submodels](https://doc.ibexa.co/projects/userguide/en/latest/personalization/recommendation_models/#submodels).
+
+#### New scenario filter
+
+Depending on a setting that you make when defining a scenario, the recommendation response can now include either product variants or base products only.
+This way you can deliver more accurate recommendations and avoid showing multiple variants of the same product to the client.
+
+For more information, see [Commerce-specific filters](https://doc.ibexa.co/projects/userguide/en/latest/personalization/filters/#commerce-specific-filters).
+
+## Other changes
+
+### Expression Language
+
+New `project_dir()` expression language function that allows you to reference current project directory.
+
+### Site Factory events
+
+Site Factory events have been moved from the `lib` directory to `contracts`.
+For a full list of events, see [Site events](https://doc.ibexa.co/en/latest/api/event_reference/site_events/).
+
+Event handling system was improved with the addition of listeners based on `CreateSiteEvent`, `DeleteSiteEvent`, and `UpdateSiteEvent`.
+New listeners automatically grant permissions to log in to a site, providing a more seamless site management experience.
+
+### Integration with Actito
+
+By using the Actito gateway you can send emails to the end-users about changes in the status of various operations in your commerce presence.
+
+### Integration with Qualifio Engage
+
+Use Qualifio Engagne integration to create engaging marketing experiences to your customers.
+
+### Integration with SeenThis!
+
+Integration with SeenThis! service provides an adaptive streaming technology with no limitations as conventional streaming service.
+It allows to preserve the best video quality with minimum amount of data transfer.
+
+For more information, see [SeenThis! block](https://doc.ibexa.co/projects/userguide/en/latest/content_management/block_reference/#sales-representative).
+
+### API improvements
+
+#### REST API
+
+##### New method signature
```

Comment:
I'd move this one to the bottom of the REST API chapter as it is really more low level.